features/projects/components/project-card/project-card.tsx

```diff
@@ -50,7 +56,9 @@ export function ProjectCard({ project }: ProjectCardProps) {
             <div className={styles.issuesNumber}>{numEvents24h}</div>
           </div>
           <div className={styles.status}>
-            <Badge color={statusColors[status]}>{capitalize(status)}</Badge>
+            <Badge color={statusColors[status]}>
+              {capitalize(statusNames[status])}
```

Comment:
No need for the capitalize here anymore as the values of the `statusNames` object are already capitablized.